pdr-api/layers/dataUtils/siteUtils.js

```diff
@@ -1,5 +1,34 @@
+const { Exception } = require('/opt/base');
 const { logger } = require('/opt/base');
-const { TABLE_NAME, runQuery } = require('/opt/dynamodb');
+const { TABLE_NAME, runQuery, getOne } = require('/opt/dynamodb');
+const { marshall } = require('@aws-sdk/util-dynamodb');
+
+const MINOR_UPDATE_TYPE = 'minor';
+const MAJOR_UPDATE_TYPE = 'major';
+const REPEAL_UPDATE_TYPE = 'repeal';
+const UPDATE_TYPES = [
+  MAJOR_UPDATE_TYPE,
+  MINOR_UPDATE_TYPE,
+  REPEAL_UPDATE_TYPE
+];
+
+const ESTABLISHED_STATE = 'established';
+const HISTORICAL_STATE = 'historical';
+const REPEALED_STATE = 'repealed';
```

Comment:
This is already exists as an exported item in dyanmodb.js.  We'll want to remove one of them.